src/coverlet.core/CoverageDetails.cs

```diff
@@ -6,9 +6,6 @@ public class CoverageDetails
     {
         public double Covered { get; internal set; }
         public int Total { get; internal set; }
-        public double Percent
-        {
-            get => Math.Round(Total == 0 ? 1 : Covered / Total, 3);
-        }
+        public double Percent => Total == 0 ? 1D : Math.Floor((Covered / Total) * 10000) / 100;
```

Comment:
should be `100D`?

Reply:
Yeah, good catch. Thanks. Will push the new commit with fix